src/disk-images/github-runner-linux/root/usr/local/bin/start-runner

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
```

Comment:
Same here should it have a copyright header?

Reply:
Added.